src/docs/community-docs/CMML/cmml-quickstart.md

```diff
@@ -117,17 +114,17 @@ It can also contain project-specific folders like 'images' or 'literature'.
 
```

Comment:
> It can also contain project-specific folders like 'images' or 'literature'.

- Maybe "additional" folders instead of "project-specific". 
- Might want to adapt the image to show the folders "images" and "literature". Although I find "images" misleading: if these images are part of sample creation (plant images -> study resources) or results (plots -> runs) there are specific places in the ARC. If they are just notes or sketches, I'd rather put them in the wiki. But I understand the incentive to collect them "inside" the ARC. 
- And maybe add them as "_images" and "_literature" or under another folder like "_communication", "_project-management", "_additional" (no good idea right now)